docs/develop/smart-contracts/evm-extensions/authorization.md

```diff
@@ -61,17 +72,34 @@ Find the [Solidity interface in the evmos/extensions repo](https://github.com/ev
         ) external returns (bool approved);
     ```
 
-### `GenericAuthorization.sol`
+### `DistributionAuthorization.sol`
 
 - `approve`
 
-    Approves a list of Cosmos message
+    Approves a list of Cosmos message.
```

Comment:
nit: definitely use plural, might make sense to also say transactions here or use messages down below to have coherence in the document.

```suggestion
    Approves a list of Cosmos transactions.
```